src/cocotb/share/lib/simulator/simulatormodule.cpp

```diff
@@ -1196,3 +1388,39 @@ PyTypeObject gpi_hdl_Object<gpi_cb_hdl>::py_type = []() -> PyTypeObject {
     type.tp_methods = gpi_cb_hdl_methods;
     return type;
 }();
+
+static PyMethodDef gpi_clk_methods[] = {
+    {"start", (PyCFunction)clk_start, METH_VARARGS,
+     PyDoc_STR("start($self, t_period, t_high, start_high)\n"
+               "--\n\n"
+               "start(t_period: int, t_high: int, start_high: bool) -> None\n"
+               "Start this clock now. The clock will have a period of "
+               "`t_period` steps, and out of that period it will be high "
+               "`t_high` steps. If `start_high` is True, start a the beginning "
+               "of the high state, otherwise start at the beginning of the "
+               "low state.\n"
```

Comment:
```suggestion
               "low state.\n"
               "\n"
```
Hopefully fixing invalid markup (https://cocotb--3983.org.readthedocs.build/en/3983/library_reference.html#cocotb.simulator.GpiClock.start)